web/src/assets/styles/blocks.scss

```diff
@@ -386,6 +403,18 @@ ul[data-type="agama/list"][role="grid"] {
   }
 }
 
+ul[data-items-type="agama/patterns"] {
```

Comment:
NOTE: this change reveals that this part, the core/Selector styles, can be simplified / improved. Let's wait a bit until selectors are more defined (we're working in new storage selectors that might have impact in such simplification if finally we get rid of storage/device-utils/DeviceSelector.